redstone/client.py

```diff
@@ -983,10 +983,35 @@ def get_instance_policies(self):
 
         https://cloud.ibm.com/apidocs/key-protect#getinstancepolicy
         """
-        resp = self.session.get("%s/api/v2/instance/policies" % (self.endpoint_url))
+        resp = self.session.get("%s/api/v2/instance/policies" % self.endpoint_url)
         self._validate_resp(resp)
         return resp.json()
 
+    def set_key_ring(self, key_id: str, key_ring_id: str, new_key_ring_id: str):
+        """
+        Transfers a key associated with one key ring to another key ring
+
+        https://cloud.ibm.com/apidocs/key-protect#patchkey
+        """
+        resp = self.session.patch(
+            "%s/api/v2/keys/%s" % (self.endpoint_url, key_id),
+            json={"keyRingID": new_key_ring_id},
+            headers={"X-Kms-Key-Ring": key_ring_id},
+        )
+        self._validate_resp(resp)
+
```

Comment:
Why do we not return response back to user when KP API does return response?

Reply:
This is base on the suggestion from @mrodden on other functions, can change to return response if we decide to.

Reply:
Currently, with the way they are written, these methods are more of a high level API which raise exceptions on errors rather than returning the raw HTTP response details. I would like to keep it that way until we need to maybe break some of the client up into high level and low level APIs, where the low level has HTTP details available.

Also, raw HTTP details will be attached to the exceptions that get raised for a user to grab additional HTTP level details to send along if needed.